executor/executor.cc

```diff
@@ -339,7 +349,9 @@ static void setup_features(char** enable, int n);
 #error "unknown OS"
 #endif
 
+#if !GOOS_windows
```

Comment:
What is the problem with including test.h on windows?

Reply:
I did not include this as of now since this is a WIP. I was thinking on making changes to test.h once we fix the bug in executor.